lab_1/main.py

```diff
@@ -3,22 +3,72 @@
 
 Count frequencies dictionary by the given arbitrary text
 """
+text = input('Enter your text: ')
```

Comment:
Provide any sample text for testing purposes. Do not use input, it breaks the check.